modules/post/windows/gather/enum_av_excluded.rb

```diff
@@ -0,0 +1,133 @@
+##
+# This module requires Metasploit: http://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+require 'msf/core'
+require 'rex'
+
+class Metasploit3 < Msf::Post
+
+  include Msf::Post::Windows::Registry
+
+  def initialize(info={})
+    super( update_info( info,
+        'Name'          => 'Windows Antivirus Excluded Locations Enumeration',
+        'Description'   => %q{ This module will enumerate all excluded directories within supported AV products },
+        'License'       => MSF_LICENSE,
+        'Author'        => [ 'Andrew Smith'],
+        'Platform'      => [ 'win' ],
+        'SessionTypes'  => [ 'meterpreter' ]
+      ))
+
+  end
+
+  def enum_mssec
+    if registry_enumkeys("HKLM\\SOFTWARE\\Microsoft").include?("Microsoft Antimalware")
+        print_status "MS Security Essentials Identified"
+        return true
+    else
+        return false
+    end
+  rescue
+    return false
+  end
+  def enum_defender
+    if registry_enumkeys("HKLM\\SOFTWARE\\Microsoft").include?("Windows Defender")
+        print_status "Windows Defender Identified"
+        return true
+    else
+        return false
+    end
+  rescue
+    return false
+  end
+
+  def enum_sep
+    if registry_enumkeys("HKLM\\SOFTWARE\\Symantec").include?("Symantec Endpoint Protection")
+        print_status "SEP Identified"
+        return true
+    else
+        return false
+    end
+  rescue
+    return false
+  end
+  def excluded_sep
+    print_status "Excluded Locations:"
```

Comment:
For every `excluded_` method where you are printing out the excluded locations found, you should probably just put this in a Rex Table with a title that includes the product name.  Running this on a system that happens to have more than one installed would result in cluttered output.